src/txn/txn_rollback_to_stable.c

```diff
@@ -111,6 +111,15 @@ __txn_abort_newer_update(
             upd->durable_ts = upd->start_ts = WT_TS_NONE;
         }
     }
+
+    /*
+     * Reset the history store flag for the stable update to indicate that this update is not
+     * written into history store, later all the aborted updates are removed from history store.
+     * Next time when this update is moved into history store, it will have different stop time
+     * pair.
```

Comment:
Minor rewording suggestion:
Clear the history store flag for the stable update to indicate that this update should not be
written into the history store later, when all the aborted updates are removed from the history store.
The next time when this update is moved into the history store, it will have a different stop time
pair.

Reply:
Fixed as suggested.